Lab1.ipynb

```diff
@@ -0,0 +1,482 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.rcParams[\"figure.figsize\"] = (15,10)\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "\n",
+    "x = np.linspace(0,10,1000);\n",
+    "prob = expon.pdf(x) \n",
+    "#ax.plot(x,prob,'r-',lw=5,alpha=0.6, label='expon pdf')\n",
+    "ax.plot(x, prob,'r-', lw=5, alpha=0.6, label='expon pdf')\n",
+    "\n",
+    "r = expon.rvs(size=100000)\n",
+    "ax.hist(r, density = True)\n",
+    "plt.show\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# A little statistis Q3A"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Selecting my hypothetical measurement value to be 1"
```

Comment:
I am a little confused here, I think the hypothetical measurement value you chose for the rest of "A little statistics part 3" is 2. See lines 182 and 184.